packages/application-shell/src/components/application-page-title/application-page-title.tsx

```diff
@@ -0,0 +1,127 @@
+import type { TEnhancedLocation } from '@commercetools-frontend/browser-history';
+
+import { useLayoutEffect } from 'react';
+import { useLocation } from 'react-router-dom';
+import startCase from 'lodash/startCase';
+import capitalize from 'lodash/capitalize';
+
+type Breadcrumb<Query extends {} = {}> = {
+  suffix: string;
+  paths: string[];
+  location?: TEnhancedLocation<Query>;
```

Comment:
I'm sorry but I don't understand why the `Breadcrumb` type includes this optional attribute since it doesn't seem is used anymore.
Perhaps I'm missing something. Could you please clarify? 🙏

Reply:
Honestly, I think I missed it, it is not used anymore after removing it from the returned object 😅
Thank you